definitions/src/lib.rs

```diff
@@ -390,19 +359,21 @@ pub enum CursorType {
 
 #[allow(non_camel_case_types)]
 #[derive(Clone, Copy, Debug, FromPrimitive)]
+#[repr(u32)]
 pub enum NoScan {
     SQL_NOSCAN_OFF = 0,
-    SQL_NOSCAN_ON,
+    SQL_NOSCAN_ON = 1,
 }
 
 #[allow(non_camel_case_types)]
 #[derive(Clone, Copy, Debug)]
+#[repr(u32)]
 pub enum BindType {
     SQL_BIND_BY_COLUMN = 0,
 }
 
 #[derive(Clone, Copy, Debug)]
-pub enum ParamOperationPtr {}
+pub enum ParamfOperationPtr {}
```

Comment:
Looks like an unwanted 'f' appeared here :D
But, shouldn't we just delete these empty enums? I don't see them used anywhere in the code